### PR TITLE
 Move CloudFlare From Domains to Security

### PR DESCRIPTION
**Pull request by jadchaar**
It seemed out of place with domain name providers.

### CODE REVIEW

**Comment by jadchaar** [comment]
Any idea why CI failed?

I was not sure how to bring multiple commits into a single patch, so I just made multiple changes and merged the patches. Could that have caused it?

**Comment by jamcat22** [comment]
You’ll need to move the `cloudflare.png` file from the ‘/img/domains/` location to the `/img/security/` location.

File: _data/security.yml [hunk 1]
```diff
@@ -40,6 +40,13 @@ websites:
       hardware: Yes
       exceptions:
           text: "Requires X.509-capable cryptotoken or smartcard."
+    
+    - name: Cloudflare
+      url: https://cloudflare.com
+      img: cloudflare.png
```
Comment (jamcat22) [inline _data/security.yml:46]: The `cloudflare.png` image needs to be moved to the `/img/security/` folder.

**Review by jamcat22 — CHANGES_REQUESTED** [review]

File: _data/security.yml (hunk 1 above)
Comment (jadchaar) [reply _data/security.yml:46]: @jamcat22 how do I do this via Github.com? I tried cloning the repo and doing it there, but I get a 403 when trying to push. Any ideas?

**Comment by jadchaar** [comment]
closing as I am opening a new PR

**Pull request closed without merging**